src/schema/posts.ts

```diff
@@ -980,7 +981,7 @@ export const resolvers: IResolvers<any, Context> = {
       await ctx.con.transaction(async (manager) => {
         const repo = manager.getRepository(Post);
         const post = await repo.findOneBy({ id });
-        if (post.type === PostType.Share) {
+        if (squadPostTypes.includes(post.type)) {
```

Comment:
I think we should not check the type here even.
If user has access to delete a post within squad, why does the type matter?

Reply:
Makes sense. I think this was just an additional check for `article` types.